src/components/GameWindow.jsx

```diff
@@ -1,29 +1,36 @@
 import React from 'react';
 import {Box} from '@material-ui/core';
-import {useLocation} from 'react-router-dom';
+import {Switch, Route} from 'react-router-dom';
 import routes from '../routes';
-import CenteringBox from './CenteringBox';
+import CenteringBox from './abs/CenteringBox';
+import MainMenu from '../pages/MainMenu';
+import Levels from '../pages/Levels';
+import MenuLayout from '../pages/MenuLayout';
 
-const GameWindow = () => {
-  const gameRoute = useLocation().pathname;
-
-  let windowContent;
-  switch (gameRoute) {
-    default:
-    case routes.MAIN:
-      windowContent = <h3>Main</h3>;
-      break;
-    case routes.LEVELSELECT:
-      windowContent = <h3>Level Select</h3>;
-      break;
-    case routes.OPTIONS:
-      windowContent = <h3>Options</h3>;
-      break;
-  }
+// TODO(Ugur):
+//    Implement different resolutions
+//    for different devices.
 
+const GameWindow = () => {
   return (
-    <CenteringBox>
-      <Box id="game-window">{windowContent}</Box>
+    <CenteringBox
+      id="game-window"
+      height="100vh"
+      width="100vw"

```

Comment:
Refer to [this stack-overflow answer for a [potentially better way of doing this.](https://stackoverflow.com/a/56254633) Also consider hiding overflows with `overflow: hidden`.